### ISSUE TITLE
Auto remove xmpp: prefix from anchor, if one was given

### ISSUE THREAD

**Issue by ge0rg** [modernxmpp/easy-xmpp-invitation#16]
Essentially do `s/^xmpp://` on the parameter.

**Comment by sjehuda** [comment]
Or using URL interface and query `hostname`, or just `location.hostname`.
```
url = new URL('xmpp:xmpp:yaxim@chat.yax.im?join');
console.log(url.hostname);
console.log(url.protocol);
console.log(url.pathname);
```

**Comment by sjehuda** [comment]
Fixed in #39
